lib/shared-ini/ini-loader.js

```diff
@@ -70,4 +72,11 @@ module.exports = AWS.util.inherit({
       new Error('Cannot load credentials, HOME path not set')
     );
   }
-});
\ No newline at end of file
+});
+
+AWS.IniLoader = IniLoader;
```

Comment:
This is still not showing up in the documentation. I believe it's because yard is looking for the pattern:
`AWS.IniLoader = AWS.util.inherit`